packages/controller-utils/src/siwe.ts

```diff
@@ -0,0 +1,53 @@
+import { ParsedMessage } from '@spruceid/siwe-parser';
+import { bytesToString, hexToBytes } from '@metamask/utils';
+
+const msgHexToText = (hex: string): string => {
```

Comment:
Suggestion: A JSDoc comment for this would be great. While it's definitely not as important for internal functions (since they don't show up in the generated API docs), they're still nice to have for the purpose of explaining what each function is meant to do. Our standard lint rules require it (though we haven't enabled that rule in this repo yet, mostly due to the backlog of functions we'd have to add docs for)

Reply:
> Our standard lint rules require it

Unfortunately it's not required for arrow functions. It definitely should be, though.

Reply:
I've switched them to regular functions and added comments now.